docker-compose.yaml

```diff
@@ -133,22 +133,21 @@ services:
       - sys_nice
     environment:
       # Self-hosted cached endpoints.
-      #- CACHED_GRPC_ENDPOINT=cached:8001
+      - CACHED_GRPC_ENDPOINT=cached:8001
```

Comment:
Only because I am curious why are we switching back to grpc here? Is something not working with the http version?

Reply:
It's mostly to align cached with yolox and paddle, which are on grpc. Besides, my understanding is that there are ongoing discussions on the performance of the http layers in the NIMs.